### ISSUE TITLE
Emit simulated events for delegate-stack-stx in PoX contract

### ISSUE THREAD

**Issue by kantai** [blockstack/stacks-blockchain#2556]
Various API consumers/event observers would like to be able to react/build around events from the PoX contract.

See:

https://github.com/blockstack/stacks-blockchain-api/issues/518

This can be addressed with some "special casing" in the `stacks-node`, which would enable a generic approach elsewhere: producing simulated events for the PoX contract for the event stream.

**Comment by kantai** [comment]
This is still something that would be useful for Stacks 2.1

**Comment by zone117x** [comment]
Is this still possible to get in the Stacks 2.1 release? It looks like there's several new PoX events that need to be added anyway.

**Comment by kantai** [comment]
Yeah, definitely. In fact, we won't have to simulate the events, we can just add new `print` statements to the `pox-2` contract.

Do you have any ideas for the different events that would be useful to emit?

**Comment by kantai** [comment]
@zone117x -- is this something that has since been solved in the event stream, or is this information still missing?

**Comment by zone117x** [comment]
This simulated event is implemented, AFAIK the remaining one is here: https://github.com/stacks-network/stacks-blockchain/issues/3529

This issue can be closed
